src/api/Feed.js

```diff
@@ -106,9 +106,10 @@ class Feed extends Base {
     setCachedItems(id: string, items: FeedItems) {
         const cache = this.getCache();
         const cacheKey = this.getCacheKey(id);
+        const sortedItems = sortFeedItems({ entries: items });
```

Comment:
Is there a reason we need to sort again after it's done in the Promise.all above?

Reply:
The add comment flow doesn't have a Promise.all that handles sorting. This setCachedItems is being called by addPendingItem on L845 which doesn't sort the feedItems